src/main/java/com/google/alpollo/SongHead.java

```diff
@@ -0,0 +1,18 @@
+package com.google.alpollo;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+
+@Entity
+public class SongHead extends Song {
```

Comment:
What is the point of this class? Why not make Song the entity directly?

Reply:
We wanted to have different classes with little information and all the information about the song. But now I have moved the id field to the parent class, so we no longer need the SongHead.